config/deployment.yml

```diff
@@ -43,19 +43,25 @@ spec:
         volumeMounts:
         - name: template-fs
           mountPath: /etc/kappctrl-mem-tmp
-        securityContext:
-          runAsUser: 1000
```

Comment:
This will default to the 1000:1000 set in the Dockerfile or be overridden on OpenShift